docs/src/pages/demos/dividers/SubheaderDividers.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Avatar from '@material-ui/core/Avatar';
+import ImageIcon from '@material-ui/icons/Image';
+import WorkIcon from '@material-ui/icons/Work';
+import BeachAccessIcon from '@material-ui/icons/BeachAccess';
+import Divider from '@material-ui/core/Divider';
+
+const styles = theme => ({
+  root: {
+    width: '100%',
+    maxWidth: 360,
+    backgroundColor: theme.palette.background.paper,
+  },
+});
+
+function SubheaderDividers(props) {
+  const { classes } = props;
+  return (
+    <div className={classes.root}>
+      <List>
+        <ListItem>
+          <Avatar>
+            <ImageIcon />
+          </Avatar>
+          <ListItemText primary="Photos" secondary="Jan 9, 2014" />
+        </ListItem>
+        <Divider subheader="Test" component="li" />
+        <ListItem>
+          <Avatar>
+            <WorkIcon />
+          </Avatar>
+          <ListItemText primary="Work" secondary="Jan 7, 2014" />
+        </ListItem>
+        <Divider variant={'inset'} subheader="Test" component="li" />
+        <ListItem>
+          <Avatar>
+            <BeachAccessIcon />
+          </Avatar>
+          <ListItemText primary="Vacation" secondary="July 20, 2014" />
+        </ListItem>
+      </List>
+    </div>
+  );
+}
```

Comment:
`fullBleed` and `inset` in the same example doesn't really make sense.

Could you use an example where it makes sense to have subheader dividers, and use a meaningful text? You could copy a portion of the example from the Google Divider spec. Or perhaps you could borrow part of this: https://material-ui.com/demos/app-bar/#bottom-app-bar (without the scrolling etc).